src/core/src/main/java/org/apache/jmeter/report/processor/graph/impl/TimeVSThreadGraphConsumer.java

```diff
@@ -54,6 +54,7 @@ public Double select(Sample sample) {
         };
     }
 
+
```

Comment:
Accidental extra line?

Reply:
Most probably, I'd thought the style check would've caught it.

Reply:
I think it only catches 3 or more.